src/pages/MyPatches.tsx

```diff
@@ -102,7 +83,7 @@ const MyPatchesComponent: React.FC = () => {
         </PageWrapper>
       );
     }
-    if (loading) {
+    if (showSkeleton) {
```

Comment:
why call it `showSkeleton` as opposed to `loading`?

Reply:
showSkeleton doesn't represent one to one the query loading state. it represents that query variables have changes & the query is loading which means the user interacted with a control on the page to adjust what data they see. 
showSkeleton is false when the query is loading because of polling bc that happens automatically from the setInterval and we don't want to display a loading skeleton every 5 seconds.